### PR TITLE
Pylama

### PR DESCRIPTION
**Pull request by tpazderka**
part 3 of #298

### CODE REVIEW

**Comment by tpazderka** [comment]
Ok, I think I am done here

File: src/oic/oic/claims_provider.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def _aggregation(self, info):
         logger.info("RESPONSE: %s" % (sanitize(cresp.to_dict()),))
         return cresp
 
-    #noinspection PyUnusedLocal
+    # noinspection PyUnusedLocal
```
Comment (lwm) [inline src/oic/oic/claims_provider.py]: I was wondering if we shouldn't just get rid of these 'no qa' comments and fix up the issues they relate to?

@rohe, you probably have something to add there?

File: src/oic/utils/userinfo/ldap_info.py [hunk 2]
```diff
@@ -86,10 +86,12 @@ def __call__(self, userid, client_id, user_info_claims=None,
...
         try:
             res = self.ld.search_s(*arg)
-        except:
+        except Exception:
+            # FIXME: This shoudl catch specific exception from `self.ld.search_s()`
             try:
                 self.ld.close()
-            except:
+            except Exception:
+                # FIXME: This shoudl catch specific exception from `self.ld.close()`
```
Comment (lwm) [inline src/oic/utils/userinfo/ldap_info.py]: shoudl -> should

File: src/oic/oic/claims_provider.py (hunk 1 above)
Comment (rohe) [inline src/oic/oic/claims_provider.py]: We should.

Comment (tpazderka) [inline src/oic/oic/claims_provider.py]: OK then, I will go through and remove those.

Comment (tpazderka) [inline src/oic/oic/claims_provider.py]: Removed from `src/*` and `tests/*`. It turned out that those issues are already resolved.

**Comment by lwm** [comment]
Great, just squash that fixup and you're good to go! :beer:

**Pull request merged**